src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationProvider.cs

```diff
@@ -249,179 +261,166 @@ public async Task RefreshAsync(CancellationToken cancellationToken)
+                        if (refreshAll)
+                        {
+                            // Trigger a single load-all operation if a change was detected in one or more key-values with refreshAll: true,
+                            // or if any key-value collection change was detected.
+                            kvEtags = new Dictionary<KeyValueSelector, IEnumerable<MatchConditions>>();
+                            ffEtags = new Dictionary<KeyValueSelector, IEnumerable<MatchConditions>>();
+                            ffKeys = new HashSet<string>();
+
+                            data = await LoadSelected(client, kvEtags, ffEtags, _options.Selectors, ffKeys, cancellationToken).ConfigureAwait(false);
+                            watchedIndividualKvs = await LoadKeyValuesRegisteredForRefresh(client, data, cancellationToken).ConfigureAwait(false);
+                            logInfoBuilder.AppendLine(LogHelper.BuildConfigurationUpdatedMessage());
+                            return;
+                        }
 
-                            if (!changedKeyValuesCollection.Any())
+                        // Get feature flag changes
+                        ffCollectionUpdated = await HaveCollectionsChanged(
+                            refreshableFfWatchers.Select(watcher => new KeyValueSelector
                             {
-                                logDebugBuilder.AppendLine(LogHelper.BuildFeatureFlagsUnchangedMessage(endpoint.ToString()));
-                            }
-                        },
-                        cancellationToken)
-                        .ConfigureAwait(false);
+                                KeyFilter = watcher.Key,
+                                LabelFilter = watcher.Label,
+                                IsFeatureFlagSelector = true
+                            }),
+                            _ffEtags,
+                            client,
+                            cancellationToken).ConfigureAwait(false);
+
+                        if (ffCollectionUpdated)
+                        {
+                            ffEtags = new Dictionary<KeyValueSelector, IEnumerable<MatchConditions>>();
+                            ffKeys = new HashSet<string>();
+
+                            ffCollectionData = await LoadSelected(
+                                client,
+                                new Dictionary<KeyValueSelector, IEnumerable<MatchConditions>>(),
+                                ffEtags,
+                                _options.Selectors.Where(selector => selector.IsFeatureFlagSelector),
+                                ffKeys,
+                                cancellationToken).ConfigureAwait(false);
+
+                            logInfoBuilder.Append(LogHelper.BuildFeatureFlagsUpdatedMessage());
```

Comment:
I don't see anywhere that we logged feature flag updates as info before. Am I missing it.

Also, why

Reply:
We had a `LogFeatureFlagUpdated` message when refreshing individual flags in the `ConfigurationClientExtensions` file, so I thought it might make sense to log the collection being updated since it seems like whenever we update a value (kv, flag, secret) we would log it as info.

Reply:
I see. In that case, I see for configuration we log "Configuration reloaded.", but for flags it is currently "Feature flags updated." It seems to me we should be consistent and log "Feature flags reloaded."

Reply:
Can you link me to where we log that today for feature flags? I still can't find it.